src/features/schedule/delete/model/useDelPostSchedule.ts

```diff
@@ -0,0 +1,26 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { deletePostSchedule } from '../api/delPostSchedule';
+import { scheduleQueryKeys } from '@/features/calendar/api/queryKeys';
+
+interface DeletePostScheduleParams {
+  postId: number;
+  scheduleId: number;
+}
+
+export const useDeletePostSchedule = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: ({ postId, scheduleId }: DeletePostScheduleParams) =>
+      deletePostSchedule(postId, scheduleId),
+
+    onSuccess: () => {
+      void queryClient.invalidateQueries({ queryKey: scheduleQueryKeys.all });
+    },
```

Comment:
게시글에 매핑된 일정을 삭제하면,
캘린더 목록 무효화, 캘린더 상세 일정 삭제, 게시글 상세 무효화 이렇게 처리해주면 어떨까요?
```suggestion
    onSuccess: (_data, { postId, scheduleId }) => {
      // 1. 캘린더 / 날짜별 일정 리스트
      void queryClient.invalidateQueries({
        queryKey: scheduleQueryKeys.lists(),
      });

      // 2. 일정 단건 캐시 제거
      queryClient.removeQueries({
        queryKey: scheduleQueryKeys.detail(scheduleId),
      });

      // 3. 게시글 상세
      void queryClient.invalidateQueries({
        queryKey: postQueryKeys.detail(postId),
      });
    },
```